acquia_lift.admin.wizard.inc

```diff
@@ -840,6 +933,9 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
     $submit_function($form, $form_state, $agent_data, $agent_instance);
   }
 
+  if (isset($form_state['values']['variations']['variation_set_handling'])) {
+    $agent_data->data['variation_set_handling'] = $form_state['values']['variations']['variation_set_handling'];
+  }
```

Comment:
I think this makes more sense in acquia_lift_personalize_campaign_wizard_variations_submit unless there is a reason it should be in the base submit handler?